radio/src/mixer_scheduler.h

```diff
@@ -58,6 +58,18 @@ uint16_t getMixerSchedulerPeriod();
 // Trigger mixer from an ISR
 void mixerSchedulerISRTrigger();
 
+// fetch the mixer schedule divider
+uint16_t getMixerSchedulerDivider();
+
+// check if internal or extern module is synced
+bool isSyncedModuleInternal();
+
+// Fetch the real scheduling period internal module
+uint16_t getRealMixerSchedulerPeriodInternal();
+
+// Fetch the real scheduling period external module
+uint16_t getRealMixerSchedulerPeriodExternal();
```

Comment:
In preparation of coming changes, I'd strongly prefer these functions to use a module ID (index) rather than explicit names. The reason is that I will start soon to make the module interface as generic as possible, without functions called `internal` or `external`, as it basically forces us to double the code or add conditional closes where not necessary.